client/src/components/Navbar/Menus/Unauthenticated/Mobile.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Mobile = () => {
+  return <div></div>;
```

Comment:
since this is just an empty div, are we planning to add anything here? 

if not, i would just remove this file and where necessary, render nothing